plugins/detabler/plugin.js

```diff
@@ -4,6 +4,8 @@
 	CKEDITOR.plugins.add('detabler', {
 		requires: 'table,contextmenu',
 		init: function (editor) {
+			var BLOCK_TAG_NAME = 'p';
```

Comment:
It's setup for something I haven't done yet, (because it doesn't matter for our settings) where it can also be `<div>`, or don't use blocks, but use `<br>`